### DISCUSSION TITLE
Support for Object Transfer Protocol

### DISCUSSION THREAD

**Discussion by lochungtin** [innoveit/react-native-ble-manager#893] [Q&A]
**Version**
 - Device: Samsung A50s
 - react-native-ble-manager version: 8.3.0
 - react-native version: 0.68.2

I am currently building a bluetooth client for a device that uses the Object Transfer Protocol and I was wondering if the library supports the use of the L2CAP channels which is required by the OTS protocol.

**Actual Behaviour**
- BleErrors while writing to characteristics and channel errors on the peripheral.

Any information regarding this issue would be of help. Thanks.

**Comment by marcosinigaglia** [comment]
Hi @lochungtin , I'm not sure the  L2CAP is part of BLE stack.
